apps/researcher/src/app/[locale]/communities/community-card.tsx

```diff
@@ -12,7 +13,15 @@ interface MembershipCountProps {
 
 async function MembershipCount({communityId, locale}: MembershipCountProps) {
   const t = await getTranslator(locale, 'Communities');
-  const memberships = await getMemberships(communityId);
+
+  let memberships = [];
+  try {
+    memberships = await getMemberships(communityId);
+  } catch (error) {
+    // This could be a sign of a deleted community in the cache.
+    // So, revalidate the communities page.
+    revalidatePath('/[locale]/communities', 'page');
```

Comment:
Is it useful to check the `error` for a specific 'the memberships have been deleted' error (e.g. HTTP status 404 or 410) before revalidating the path?

Reply:
Yes. During testing the response was always `404`. Just to be sure I will also check for `410`